### ISSUE TITLE
docker container / installation log for debian

### ISSUE THREAD

**Issue by nagimov** [agendav/agendav#243]
I've just spent couple evenings compiling a recipe for agendav docker image, available [here](https://github.com/nagimov/agendav-docker)
It's useful as is, but also can be used as an installation log, if you look at the [Dockerfile](https://github.com/nagimov/agendav-docker/blob/master/Dockerfile) - applicable to debian/ubuntu

**Comment by MartenBE** [comment]
I am trying to configure this to run together with a docker instance of radicale on a single machine. The radicale docker runs (I can see it on localhost:5232 and login), but how do I run this docker to connect to that?

What I've done:
```
podman run -d --name radicale -p 5232:5232 tomsquest/docker-radicale
podman run -d --name agendav -p 5233:80 -e AGENDAV_CALDAV_SERVER="127.0.0.1:5232" -e AGENDAV_SERVER_NAME="127.0.0.1" nagimov/agendav-docker
```

I get a blank page at localhost:5233.

**Comment by nagimov** [comment]
Many clients do not support the creation of new calendars and address books. You can use Radicale’s web interface (e.g. http://localhost:5232) to create and manage collections.
In some clients you can just enter the URL of the Radicale server (e.g. http://localhost:5232) and your user name. In others, you have to enter the URL of the collection directly (e.g. http://localhost:5232/user/calendar).

see more at https://radicale.org/clients/

**Comment by MartenBE** [comment]
I've made it work thanks to using the `--network="host"` option in the 2nd `podman run` command. It would be cleaner by using a docker-compose file, but for now it works.

**Comment by pixelbrackets** [comment]
Would be good to add a link to the image in the docs.

**Comment by nagimov** [comment]
@pixelbrackets thanks for cleaning things up! :+1: 
#287 updates readme with a link to docker hub. Docker image is just updated to latest debian.
Is 2.4.0 stable enough to point the `latest` docker tag to it?

**Comment by nagimov** [comment]
discussion in PR #287

**Comment by pixelbrackets** [comment]
Resolved in PR #287, a link to the image is [included in the README](https://github.com/agendav/agendav/tree/development#docker-image). Thanks for your work on this! :100: :medal_sports:
